docs/concepts/providers.md

```diff
@@ -7,41 +7,34 @@ indent: true
 
 # Providers
 
-Providers extend Crossplane to enable infrastructure resource provisioning. In
-order to provision a resource, a Custom Resource Definition (CRD) needs to be
-registered in your Kubernetes cluster and its controller should be watching the
-Custom Resources those CRDs define. Provider packages contain many Custom
-Resource Definitions and their controllers.
+Providers are Crossplane packages that bundle a set of [Managed
+Resources][managed-resources] and their respective controllers to allow
+Crossplane to provision the respective infrastructure resource.
 
 Here is the list of prominent providers:
 
-### AWS Provider
+|   |   |   |   |
```

Comment:
Wonder if it would be worth adding headers here for each column -- they are pretty self-explanatory, but the auth mechanism part is somewhat confusing (I do realize these are persisted from the original content though). Any thoughts on how we could make this clearer? Might be worth requiring an `AUTHENTICATION.md` or similar in each repo.

Reply:
I don't see a big issue with the current table TBH. I don't know if an Authentication.md is really needed for every repo. And if it does we should do it in a new repo.

Reply:
@luebken fair assessment -- I would like for us to have some more standardization on how folks discover authentication methods, but I don't think this PR is necessarily the place to accomplish that 👍🏻